spec/views/api_new/payouts/show.json.jbuilder_spec.rb

```diff
@@ -5,7 +5,7 @@
 require 'rails_helper'
 
 RSpec.describe '/api_new/payouts/show.json.jbuilder', type: :view do
-  let(:payout) { build(:payout) }
+  let(:payout) { build(:payout, nonprofit: build(:nonprofit)) }
```

Comment:
This file needs to check that status is added to the payout JSON as expected in the test between line 17-23